src/docs/reference/modules/ReCaptcha/README.md

```diff
@@ -0,0 +1,32 @@
+# ReCaptcha
+
+The OrchardCore.ReCaptcha module can be used to prevent robots access to your OrchardCore website.
```

Comment:
'to prevent from accessing your' may be.

Reply:
done